lib/rpc/rpcclient.js

```diff
@@ -5,6 +5,8 @@ var guid = require('node-uuid').v4;
 var commUtil = require('../util');
 var manifestLoader = require('./manifest-loader');
 
+var DEFAULT_TIMEOUT = 20000;
+
```

Comment:
Maybe move this to a file like `rpc/common.js` that way we don't have it declared twice. May as well while we're changing in.